src/SignInComponent.tsx

```diff
@@ -0,0 +1,342 @@
+    display: inline-block;
+`
+
+export const smallRightSpacer = css`
+    margin-right: ${space[3]}px;
+`
+
+export const primaryButton = css `
+    margin-right: ${space[3]}px;
+    background-color: #333;
+    color: #fff;
+    &:hover {
+        background-color: #111;
+    }
+`
+
+export const secondaryButton = css `
+    color: #333;
+`
+
+export const packShot = css`
+    max-width: 100%;
+    max-height: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: flex-end;
+    margin-top: -20px;
+    img {
+        max-width: 100%;
+        max-height: 100%;
+        object-fit: contain;
+    }
+    ${until.tablet} {
+        display: none;
+    }
+`;
+
+export const iconPanel = css`
+    ${from.desktop} {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        align-items: flex-end;
+        height: 100%;
+        padding: ${space[4]}px 0;
+        margin-left: ${space[4]}px;
+    }
+`;
+
+
+export const closeButton = css`
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 0;
+    border: 1px solid ${palette.text.primary};
+    border-radius: 50%;
+    outline: none;
+    background: transparent;
+    cursor: pointer;
+    width: 35px;
+    height: 35px;
+    svg {
+        width: 25px;
+        height: 25px;
+        fill: ${palette.text.primary};
+        transition: background-color 0.5s ease;
+        border-radius: 50%;
+    }
+    :hover {
+        cursor: pointer;
+        background-color: rgba(237, 237, 237, 0.5);
+    }
+    
+    position: absolute;
+    top: 10px;
+    right: 10px;
+`;
+
+export const smallInfoIcon = css`
+    svg {
+        width: 40px;
+        height: 40px;
+        vertical-align: -30%;
+        fill: #CDCDCD;
+    }
+    left: -60px;
+    display: none;
+    ${until.tablet}{
+        display: block;
+        margin-left: -4px;
+        margin-top: -4px;
+        margin-bottom: 4px;
+    }
+
+`;
+
+export const infoIcon = css`
+    svg {
+        width: 60px;
+        height: 60px;
+        fill: #CDCDCD;
+    }
+    position: absolute;
+    width: 60px;
+    height: 60px;
+    left: -60px;
+    top: 10px;
+    display: none;
+    @media (min-width: 1430px){
+        display: block;
+    }
+`;
+
+export const storeIcon = css`
+    height: 10px;
+    svg {
+        height: 30px;
+        width: auto;
+        margin-top: ${space[2]}px;
+        padding-right: ${space[2]}px;
+    }
+    display: inline-block;
+    vertical-align: -37.5%;
+    
+`;
+
+export const SignInComponent: React.FC<Props> = ({
+  message,
+  onButtonClick,
+  firstName
+}: Props) => {
+    const [showBanner, setShowBanner] = useState(true);
+
+    const onCloseClick = (evt: React.MouseEvent<HTMLButtonElement, MouseEvent>, buttonId: string): void => {
+        evt.preventDefault();
+        onButtonClick(buttonId);
+        setShowBanner(false);
+    };
+
+    return (
+        <>
+            {showBanner ? (
```

Comment:
What do you think about replacing this ternary with an early return when `showBanner` is false? I think that would be slightly more clear/readable.

Reply:
Yes good idea, I'll tidy it up a bit.